src/Symfony/Component/ErrorRenderer/Exception/FlattenException.php

```diff
@@ -37,6 +37,16 @@ class FlattenException
     private $file;
     private $line;
 
+    /**
+     * @deprecated since Symfony 4.4, use createFromThrowable() instead.
+     */
+    public static function create(\Exception $exception, $statusCode = null, array $headers = [])
+    {
+        @trigger_error(sprintf('The "%s()" method is deprecated since Symfony 4.4, use "createFromThrowable()" instead.', __METHOD__), E_USER_DEPRECATED);
+
+        return static::createFromThrowable($exception, $statusCode, $headers);
+    }
+
     public static function createFromThrowable(\Throwable $exception, int $statusCode = null, array $headers = []): self
```

Comment:
what about renaming this one to `create` instead? that would make the most sense to me as the current name is due to history - but this is a new component now

Reply:
But if somebody did use `createFromThrowable` which did also exist on the Debug FlattenException it will get there an error. I think in 4.4 we still should provide both functions.